geoips/plugins/yaml/product_defaults/georing_3d/Cloud_Base_Height.yaml

```diff
@@ -0,0 +1,25 @@
+interface: product_defaults
+family: algorithm_interpolator_colormapper
+name: Cloud_Base_Height
+docstring: |
```

Comment:
Should these be in their own plugin package repo?  Or combined with geoips_clavrx?  Is there some duplication here?